INFO     [🔧TEST CONFIG]:conftest.py:87 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:93 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:123 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:109 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:117 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:205 Continue using UserData(email='regression-b3ava4qwop@apolo.us', username='regression-b3ava4qwop', password='0UmOYmbvHYYl') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-b3ava4qwop@apolo.us', username='regression-b3ava4qwop', password='0UmOYmbvHYYl')
INFO     [🔧TEST CONFIG]:conftest.py:235 Continue using UserData(email='regression-dkyztxdzdt@apolo.us', username='regression-dkyztxdzdt', password='z1H#A63NIGkj') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-dkyztxdzdt@apolo.us', username='regression-dkyztxdzdt', password='z1H#A63NIGkj')
INFO     [🔧TEST CONFIG]:conftest.py:240 Continue using UserData(email='regression-xw8tcsgzjh@apolo.us', username='regression-xw8tcsgzjh', password='#ultBr^5f74D') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-xw8tcsgzjh@apolo.us', username='regression-xw8tcsgzjh', password='#ultBr^5f74D')
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify Apolo CLI client is installed
INFO     [🖥apolo_CLI]:apolo_runner.py:25 check CLI version. Running command via cli:

apolo --version

INFO     [🖥apolo_CLI]:apolo_runner.py:86 check CLI version succeeded:

Apolo Platform Client 25.8.1

INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify Apolo CLI client is installed
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: Admin removes Manager from org via **CLI**
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_4aba4d.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-b3ava4qwop@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_67679d.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_83ba94.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception ClientResponseError: 500, message='Internal Server Error', url='http://platform-admin:8080/apis/admin/v1/users/regression-b3ava4qwop?include=orgs&include=clusters&include=projects'. Path with query: /api/v1/config."}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-b3ava4qwop/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[]
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Timeout error for spinner
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_f46bf5.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_fail_aa20cf.png
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_fail_7c016e.png
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: Admin removes Manager from org via **CLI**
Exception: [AssertionError] during step: Login via UI
Message: [AssertionError] during step: Verify that Welcome new user page displayed
Message: Locator expected to be visible
Actual value: <element(s) not found> 
Call log:
  - Expect "to_be_visible" with timeout 5000ms
  - waiting for locator("h3.truncate.text-h3").filter(has_text="Welcome, regression-b3ava4qwop@apolo.us")

INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:54 GET https://api.dev.apolo.us/apis/admin/v1/orgs
INFO     [🌐API_helper]:api_helper.py:165 Status: 200. Response: []
INFO     [🔧TEST CONFIG]:conftest.py:291 Cleaning up 0 organisations
INFO     [🔧TEST CONFIG]:conftest.py:321 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:347 Browser cleanup finished
INFO     [🔧TEST CONFIG]:conftest.py:87 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:93 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:123 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:109 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:117 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:205 Continue using UserData(email='regression-b3ava4qwop@apolo.us', username='regression-b3ava4qwop', password='0UmOYmbvHYYl') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-b3ava4qwop@apolo.us', username='regression-b3ava4qwop', password='0UmOYmbvHYYl')
INFO     [🔧TEST CONFIG]:conftest.py:235 Continue using UserData(email='regression-dkyztxdzdt@apolo.us', username='regression-dkyztxdzdt', password='z1H#A63NIGkj') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-dkyztxdzdt@apolo.us', username='regression-dkyztxdzdt', password='z1H#A63NIGkj')
INFO     [🔧TEST CONFIG]:conftest.py:240 Continue using UserData(email='regression-xw8tcsgzjh@apolo.us', username='regression-xw8tcsgzjh', password='#ultBr^5f74D') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-xw8tcsgzjh@apolo.us', username='regression-xw8tcsgzjh', password='#ultBr^5f74D')
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify Apolo CLI client is installed
INFO     [🖥apolo_CLI]:apolo_runner.py:25 check CLI version. Running command via cli:

apolo --version

INFO     [🖥apolo_CLI]:apolo_runner.py:86 check CLI version succeeded:

Apolo Platform Client 25.8.1

INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify Apolo CLI client is installed
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: Admin removes Manager from org via **CLI**
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_ba330c.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-b3ava4qwop@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_9071a7.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_2b5890.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": true, "orgs": [], "clusters": [], "projects": [], "admin_url": "https://api.dev.apolo.us/apis/admin/v1", "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner to disappear
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-b3ava4qwop/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[]
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/invites/incoming
Request Body:
<not available>
Response Body (first 1000 chars):
[]
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-b3ava4qwop?include=orgs
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-b3ava4qwop',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/users/regression-b3ava4qwop?include=orgs."}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-b3ava4qwop?include=clusters&include=system_role
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-b3ava4qwop',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/users/regression-b3ava4qwop?include=clusters&include=system_role."}
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Timeout error for spinner
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_c6f8ed.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_fail_d89656.png
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_fail_403b11.png
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: Admin removes Manager from org via **CLI**
Exception: [AssertionError] during step: Login via UI
Message: [AssertionError] during step: Verify that Welcome new user page displayed
Message: Locator expected to be visible
Actual value: <element(s) not found> 
Call log:
  - Expect "to_be_visible" with timeout 5000ms
  - waiting for locator("h3.truncate.text-h3").filter(has_text="Welcome, regression-b3ava4qwop@apolo.us")

INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:54 GET https://api.dev.apolo.us/apis/admin/v1/orgs
INFO     [🌐API_helper]:api_helper.py:165 Status: 200. Response: []
INFO     [🔧TEST CONFIG]:conftest.py:291 Cleaning up 0 organisations
INFO     [🔧TEST CONFIG]:conftest.py:321 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:347 Browser cleanup finished
INFO     [🔧TEST CONFIG]:conftest.py:87 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:93 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:123 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:109 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:117 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:205 Continue using UserData(email='regression-b3ava4qwop@apolo.us', username='regression-b3ava4qwop', password='0UmOYmbvHYYl') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-b3ava4qwop@apolo.us', username='regression-b3ava4qwop', password='0UmOYmbvHYYl')
INFO     [🔧TEST CONFIG]:conftest.py:235 Continue using UserData(email='regression-dkyztxdzdt@apolo.us', username='regression-dkyztxdzdt', password='z1H#A63NIGkj') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-dkyztxdzdt@apolo.us', username='regression-dkyztxdzdt', password='z1H#A63NIGkj')
INFO     [🔧TEST CONFIG]:conftest.py:240 Continue using UserData(email='regression-xw8tcsgzjh@apolo.us', username='regression-xw8tcsgzjh', password='#ultBr^5f74D') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-xw8tcsgzjh@apolo.us', username='regression-xw8tcsgzjh', password='#ultBr^5f74D')
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify Apolo CLI client is installed
INFO     [🖥apolo_CLI]:apolo_runner.py:25 check CLI version. Running command via cli:

apolo --version

INFO     [🖥apolo_CLI]:apolo_runner.py:86 check CLI version succeeded:

Apolo Platform Client 25.8.1

INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify Apolo CLI client is installed
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: Admin removes Manager from org via **CLI**
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_d2f3f0.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-b3ava4qwop@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_26bd87.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_a01d69.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": true, "orgs": [], "clusters": [], "projects": [], "admin_url": "https://api.dev.apolo.us/apis/admin/v1", "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner to disappear
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-b3ava4qwop/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[]
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/invites/incoming
Request Body:
<not available>
Response Body (first 1000 chars):
[]
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-b3ava4qwop?include=orgs
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-b3ava4qwop',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/users/regression-b3ava4qwop?include=orgs."}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-b3ava4qwop?include=clusters&include=system_role
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-b3ava4qwop',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/users/regression-b3ava4qwop?include=clusters&include=system_role."}
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Timeout error for spinner
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_361e12.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_fail_f43d9a.png
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_fail_397d05.png
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: Admin removes Manager from org via **CLI**
Exception: [AssertionError] during step: Login via UI
Message: [AssertionError] during step: Verify that Welcome new user page displayed
Message: Locator expected to be visible
Actual value: <element(s) not found> 
Call log:
  - Expect "to_be_visible" with timeout 5000ms
  - waiting for locator("h3.truncate.text-h3").filter(has_text="Welcome, regression-b3ava4qwop@apolo.us")

INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:54 GET https://api.dev.apolo.us/apis/admin/v1/orgs
INFO     [🌐API_helper]:api_helper.py:165 Status: 200. Response: []
INFO     [🔧TEST CONFIG]:conftest.py:291 Cleaning up 0 organisations
INFO     [🔧TEST CONFIG]:conftest.py:321 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:347 Browser cleanup finished